index.js

```diff
@@ -15,6 +15,11 @@ module.exports = function (processors, options) {
 
   stream._transform = function (file, encoding, cb) {
 
+    if (file.isNull()) {
+      this.push(file);
+      return cb();
```

Comment:
Can be written like this: `return cb(null, file)`. 

Please adjust to the coding style (no semicolons) and add a test.